Dockerfile

```
@@ -0,0 +1,35 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /ros2_workspace/src
```

```suggestion
WORKDIR /root/ros2_ws/src
```

Can we match typical environments from the ROS tutorials which are in the home directory?